operations/system_management_health/Grafterm.md

```diff
@@ -0,0 +1,57 @@
+# Grafterm
+
+Visualize metrics dashboards on the terminal, like a simplified and minimalist version of **Grafana** for terminal.
+
+The utility(script) can be found in the /opt/cray/platform-utils directoty in all NCNs.
+
+## Running options
+
+Exit with `q` or `Esc`
+
+### Help
+
+```bash
+ncn-m001# ./grafterm.sh --help
+```
+
+### List available terminal dashboards
+
+```bash
+ncn-m001# ./grafterm.sh --list
+```
+
+### simple
```

Comment:
Maybe 'Default Usage'?  Could also say what the default days and refresh settings are...

Reply:
> Maybe 'Default Usage'? Could also say what the default days and refresh settings are...

Addressed. Thanks!